docs/sources/developers/plugins/add-support-for-explore-queries.md

```diff
@@ -1,22 +1,18 @@
 ---
-title: Add support for Explore queries
+title: Add features to Explore queries
 ---
 
-# Add support for Explore queries
+# Add features to Explore queries
```

Comment:
created https://github.com/grafana/grafana/issues/68148 to track

Reply:
Is 'features **to** Explore' still preferred to 'features **for** Explore'? Would rather limit the number of times we change page titles to avoid any redirect pain, so just checking as this is separate to the comment about improving the page generally.